src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -877,6 +877,23 @@ const schema = {
         },
       },
     },
+    accountType: {
+      type: 'string',
+      enum: ['Checking', 'Savings'],
+    },
+    accountNumber: {
+      type: 'string',
+      minLength: 4,
+      maxLength: 17,
+    },
+    routingNumber: {
+      type: 'string',
```

Comment:
I think I've hunted down this rabbit before, but do we have a `number` type we could use that tells the `StringField` the input should be `type="number"`?

Reply:
Unfortunately switching to `type: 'number'` means we can't use `minLength`, `maxLength`, or `pattern` validations so this isn't really viable given the restrictions we need to place on the input.